packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -39,12 +36,18 @@ export default class TooltipBubble extends React.Component<Props> {
         // gain over just using a div.
         return (
             <div
```

Comment:
Is the extract lifting calling `findDOMNode` or is there other work that's involved?

Reply:
Oh wait, it's a typo: "extra lifting". OK, yeah, so this actually goes away in #159. I moved to a `View` there and use `findDOMNode`